src/server.rs

```diff
@@ -406,10 +439,21 @@ async fn index(
     );
 
     // After middleware
-    response = match middleware_router.get_route(&MiddlewareRoute::AfterRequest, req.uri().path()) {
+    // Global
+    response = middleware_router
+        .get_global_middlewares(&MiddlewareType::AfterRequest)
+        .iter()
+        .fold(response, |response, function| {
+            block_on(execute_middleware_function(&response, function)).unwrap_or_else(|e| {
+                error!("Error while executing after middleware: {e}");
+                response
+            })
+        });
```

Comment:
nicee. we can extend it to middleware chaining in the future 😄

Reply:
Yes I figured that too ^^ I could try to build onto this work in later PRs to implement middleware chaining ^^